src/lib/analysers/ProtestsPretrained/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytorch
+pillow
```

Comment:
An explicit version for pillow would also be better. 

Mtriage doesn't deal _that_ well with incompatible versions between analysers yet... but it's in the works, and so explicit labelling in component requirements is ideal.